source/documentation/deploying-an-app/pod-resource-general-guideline.html.md.erb

```diff
@@ -0,0 +1,143 @@
+---
+title: General Guidelines for Pod Requests and Limits
+last_reviewed_on: 2024-06-27
+review_in: 6 months
+layout: google-analytics
+---
+
+# <%= current_page.data.title %>
+
+To ensure the stability, scalability, and efficient use of resources within our Kubernetes cluster, it is important to follow specific guidelines when configuring resource requests and limits for pods. 
+This section provides an a guideline to set up resource requests and limits for pods and to maintain an optimal cluster state.
+
+## Understand Resource Requests and Limits
+
+- Requests: The amount of CPU and memory guaranteed to a container. These are the minimum resources your application needs to run efficiently.
+
+- Limits: The maximum amount of CPU and memory a container is allowed to use. These prevent a single container from consuming excessive resources and impacting other applications.
```

Comment:
```suggestion
- Limits: The maximum amount of CPU and memory a container is allowed to use. These prevent a single container from consuming excessive resources and impacting other applications.
```

Is it worth adding some detail here about how in favour of setting high limits, you should aim to instead look at scaling up your replicas. Limit values are not calculated by the Kubernetes scheduler and so cannot be used as a guarantee of resource depending on the node capacity.

Reply:
Added Scaling and Resource Limits session